tasks/move-granules/tests/index-test.js

```diff
@@ -160,33 +160,71 @@ test.serial('should overwrite files', async (t) => {
     name: filename
   }];
 
+  const uploadParams = {
+    Bucket: t.context.stagingBucket,
+    Key: sourceKey,
+    Body: 'Something'
+  };
+
+  console.log(`CUMULUS-970 debugging: start s3 upload. params: ${JSON.stringify(uploadParams)}`);
+
   await promiseS3Upload({
```

Comment:
Might as well just pass in `uploadParams` here

Reply:
done